crates/bdk/src/wallet/mod.rs

```diff
@@ -815,6 +815,8 @@ impl<D> Wallet<D> {
     }
 
     /// Return whether or not a `script` is part of this wallet (either internal or external)
+    /// Use `derivation_of_spk` function to find the `KeychainKind` for the `script`, or
+    /// None if there's no  associated `Keychain`.
```

Comment:
Please follow rust doc conventions. Ref: https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components

We want a separate section for the more detailed description. Also, for clarity, I would reword it like so: "If you wish to find the `KeychainKind` for the given `script`, use `derivation_of_spk` instead."

In addition to this, please have a link for `derivation_of_spk`.

Reply:
Would it make sense to have  seperate section telling about the  `derivation_of_spk` in detail  and include that point ?

Reply:
If you want to expand on `derivation_of_spk`, it should be part of the docs on `derivation_of_spk`, not here.